JenkinsTransport/Api.cs

```diff
@@ -173,7 +173,7 @@ public ProjectStatus GetProjectStatus(XDocument xDoc, ProjectStatus currentStatu
                                                   lastCompletedBuildInfo.Timestamp)
                                     {
                                         Activity = EnumUtils.GetProjectActivity(color),
-                                        Status = EnumUtils.GetProjectIntegratorState((bool)firstElement.Element("buildable")),
+                                        Status = EnumUtils.GetProjectIntegratorState(((string)firstElement.Element("buildable"))?.EndsWith("rue") ?? false),
```

Comment:
Please don't add complexity by putting null checks on more logic inline here, it ends up increasing the cognitive load of this line of code.

instead extract this to a private method

bool buildable = GetBuildableStatus(firstElement.Element("buildable"));

private bool Buildable(string buildable)
{
    if (buildable == null)
        return false;

   if (buildable != "true")
       return false;

   return true;
}


That kind of code is much easier to unpack and read at a glance. It's not about how small you can make the lines of code, the compiler does not care ;-)

Then just do

Status = EnumUtils.GetProjectIntegratorState(buildable);